### PR TITLE
Pr code enhancements

### PR DESCRIPTION
**Pull request by larsk21**
- rename TimeSheet variables to timeSheet
- remove magic numbers in WorkingArea
- remove magic numbers in TimeSpan

### CODE REVIEW

**Comment by larsk21** [comment]
If you have additional issues for this PR, please add them before approving.

File: src/main/java/data/TimeSpan.java [hunk 1]
```diff
@@ -5,7 +5,15 @@
  */
 public class TimeSpan implements Comparable<TimeSpan> {
     
-    //TODO Replace magic numbers with constants
+    public static final int MIN_HOUR = 0;
+    public static final int MIN_MINUTE = 0;
+    public static final int MAX_MINUTE = 59;
```
Comment (gstuer) [inline src/main/java/data/TimeSpan.java]: Are these constant values public for a specific reason?

Comment (larsk21) [reply src/main/java/data/TimeSpan.java]: Yes, so that the user of this class knows what values the constructor is allowed to call with.

Comment (gstuer) [reply src/main/java/data/TimeSpan.java]: That's fine! It just wasn't clear for me at the first moment but I see what your intention was :)

**Pull request merged**